addons/godot-xr-tools/staging/scene_base.gd

```diff
@@ -22,40 +22,41 @@ signal load_scene
 
 ## Interface
 
+func _ready() -> void:
+	pass
 
 # Add support for is_xr_class on XRTools classes
 func is_xr_class(name : String) -> bool:
 	return name == "XRToolsSceneBase"
 
-
 func center_player_on(p_transform : Transform3D):
 	# In order to center our player so the players feet are at the location
 	# indicated by p_transform, and having our player looking in the required
 	# direction, we must offset this transform using the cameras transform.
-	
+
```

Comment:
Totally unrelated to this PR but maybe we should add the pblank scripts from Godot to our plugin, or @Malcolmnixon would the linter take care of this if we go through the pain of cleaning everything up for it?

Reply:
Ah, yeah, sorry about this! vscode likes to remove whitespace from lines that have only whitespace